### DISCUSSION TITLE
Option to Disable Update Pop-ups

### DISCUSSION THREAD

**Discussion by Jishnuraj9** [ente-io/ente#3288] [Enhancements]
Currently, the app displays pop-ups related to updates, which redirect to GitHub. This can be disruptive, especially when trying to quickly access codes or perform other tasks within the app.

**Comment by Damariobros** [comment]
Perhaps a "don't show this message again" button or checkbox could work? Then when the person is ready to update on their own terms, they can use the "check for updates" button in the Settings menu to get the link to the GitHub.

Once Ente is actually finally updated the option could turn off, so the next time a new update comes out they'll get notified, but Ente could remember that it was checked before in the previous installation and autofill the checkbox so they can quickly dismiss it again, maybe that could be a good compromise.
